pokete.py

```diff
@@ -1101,26 +1145,27 @@ def add(self):
 
 class Buy:
     """Menu to buy items in, is triggered in shop"""
+
     def __init__(self):
-        self.box = ChooseBox(height-3, 35, "Shop")
-        self.box2 = Box(7, 21,)
+        self.box = ChooseBox(height - 3, 35, "Shop")
+        self.box2 = Box(7, 21, )
```

Comment:
```suggestion
        self.box2 = Box(7, 21)
```
This comma can be removed.